ui-admin/src/forms/designer/NewQuestionForm.tsx

```diff
@@ -17,42 +19,6 @@ export const NewQuestionForm = (props: NewQuestionFormProps) => {
   const [selectedQuestionType, setSelectedQuestionType] = useState<QuestionType>('text')
   const [questionName, setQuestionName] = useState<string>('')
```

Comment:
This looks like duplicated state to me. I think this could be replaced with `const { name: questionName } = question` (below the `useState<Question>` line).

Reply:
Good eye, fixed in 0ecef11f